pre_built_networks/1class8/secondary_training_config.json

```diff
@@ -0,0 +1,63 @@
+{
```

Comment:
Can this file be moved to the root directory? It applies to the model in general, right?